Extensions/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/basicSearchSettings/index.jsx

```diff
@@ -54,34 +47,6 @@ class BasicSearchSettingsPanelBody extends Component {
         }
     }
 
-    componentDidUpdate(props) {

```

Comment:
I am also concerned about this removal without it being implemented another way. Although I also don't like the original implementation, I am scared to just remove it without seing it reimplemented another way. Also since the tab that has this does not load currently, I have no way to property test the effects of this change.

Reply:
Here it can be removed since it's redundant and misused. All we need already happens in **onSettingChange()** method.